src/main.cpp

```diff
@@ -190,14 +216,76 @@ int main(const int argc, const char *argv[])
     // Shift insets so that they do not overlap
     cart_info.shift_insets_to_target_position();
 
+    // Scale equal area map up
+    double scale_factor = 10e3;
+    for (auto &[inset_pos, inset_state] : *cart_info.ref_to_inset_states()) {
+      inset_state.scale_inset(scale_factor);
+    }
+
     // Output to GeoJSON
     cart_info.write_geojson(
       geo_file_name,
       map_name + "_equal_area.geojson",
-      output_to_stdout);
+      output_to_stdout,
+      insert_visual_variable);
     return EXIT_SUCCESS;
   }
 
+  // Compare 2 GeoJSON files.
+  if (compare_geo_file_name != "") {
+    CartogramInfo compare_cart_info(world, visual_file_name);
+    init_cart_info(&compare_cart_info,
+                    arguments,
+                    make_csv,
+                    world,
+                    simplify,
+                    output_equal_area,
+                    target_points_per_inset,
+                    compare_geo_file_name,
+                 map_name);
+
```

Comment:
This might be over-complication, since it's just debug output. Maybe it's a good idea to output to a CSV file, but another PR is more appropriate for that I think